### DISCUSSION TITLE
Logs in the dashboard do not close (1.34.0)

### DISCUSSION THREAD

**Discussion by richimaint** [Koenkk/zigbee2mqtt#19959] [General]
These clues do not close automatically?
![grafik](https://github.com/Koenkk/zigbee2mqtt/assets/91455484/8772c0d2-81ff-4430-9e12-009deb928e53)

Z2M V 1.34.0

richimaint

**Comment by julianfs** [comment]
Click each one to close

**Comment by richimaint** [comment]
I know, but before 1.34.0 it closed itself

**Comment by DaN66O** [comment]
Actually, this is a bug in z2m.
Is it going to be fixed or is it meant to be?

**Comment by Koenkk** [comment]
Will be fixed in https://github.com/nurikk/zigbee2mqtt-frontend/issues/1847
